cmd/apiserver/app/server/run_server.go

```diff
@@ -49,8 +49,10 @@ func runTPRServer(opts *ServiceCatalogServerOptions) error {
 	glog.Infoln("Installing TPR types to the cluster")
 	tprInstaller := tpr.NewInstaller(tprOpts.clIface.Extensions().ThirdPartyResources())
 	if err := tprInstaller.InstallTypes(); err != nil {
-		glog.V(4).Infof("Installing TPR types failed, continuing anyway (%s)", err)
+		glog.Errorf("Failed to installing TPR types (%s)", err)
+		return err
```

Comment:
@simonleung8 I have a nit here: can you return a well-known error -- either an exported error type or exported error variable -- so that you can reliably check the type or value in `TestRunServerInstallTPRFails`, instead of the error string?